.github/workflows/on-pr-to-main.yml

```
@@ -0,0 +1,40 @@
+name: Test Notebooks on PR to Main
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)